static/src/js/components/SpatialSelection/SpatialSelection.jsx

```diff
@@ -529,6 +529,23 @@ const SpatialSelection = (props) => {
   // Callback from EditControl, called when the controls are mounted
   const onMounted = (newDrawControl) => {
     drawControl.current = newDrawControl
+
+    // Add aria-labels to draw buttons
+    const drawToolbar = document.querySelector('.leaflet-draw-toolbar')
+    if (drawToolbar) {
+      const buttons = drawToolbar.querySelectorAll('.leaflet-draw-draw-circle, .leaflet-draw-draw-polygon, .leaflet-draw-draw-rectangle, .leaflet-draw-draw-marker')
+      buttons.forEach((button) => {
+        if (button.classList.contains('leaflet-draw-draw-circle')) {
+          button.setAttribute('aria-label', 'Search by spatial circle')
```

Comment:
These should read "Draw a [spatial method] on the map to select a spatial extent"